src/ray/python/default_worker.py

```diff
@@ -0,0 +1,14 @@
+import argparse
+
+from worker import Worker
+
```

Comment:
also, all Python files need to have

```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

at the top

Reply:
this worker is to be replaced with the real worker ported from master. Do we need to spend time on grooming this one?

Reply:
I've already added these imports.